src/install/search.rst

```diff
@@ -33,7 +33,16 @@ Installation of Binary Packages
 
 Binary packages that bundle all the necessary dependencies of the search plugin are
 available on `GitHub`_.  The files in each release should be unpacked into a directory on
-the Java classpath. The service expects to find a couple of configuration files
+the Java classpath. If you do not have a classpath already set, or you wish to explicitly
+set the classpath location for Clouseau, then add the line::
+
+-classpath '/path/to/clouseau/*'
```

Comment:
Please indent this line by 4 spaces to ensure it is correctly formatted.

Reply:
I may have misunderstood as I just accepted the suggestion. Which line should be indented?

Reply:
Lines 39 and 43 need to be indented by 4 spaces.

Reply:
All done.